src/schemas/xspec.rnc

```diff
@@ -13,13 +12,21 @@ description =
 	## It may also have a version to aid development over time.
 	element description { 
 		common-attributes,
-		attribute stylesheet { xs:anyURI }?,
-		attribute query { xs:anyURI }?,
-		attribute xslt-version { xs:decimal }?,
-		attribute query-at { xs:anyURI }?,
-		attribute version { xs:decimal }?,		
-		attribute preserve-space { xs:NMTOKENS }?,
-		attribute schematron { xs:anyURI }?,
+		attribute stylesheet { xsd:anyURI }?,
+		attribute query { xsd:anyURI }?,
+
+		## XQuery version which will be declared in the compiled main module.
+		attribute xquery-version { text }?,
```

Comment:
Your description explained why you named this xquery-version instead of query-version. I think it makes perfect sense to name it xquery-version because it's the version of the language, not the version of the query file being tested.

Reply:
Thanks for the review. I'm merging this pr.